### PR TITLE
xk6, Clarify output extensions

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**

### CODE REVIEW

File: src/data/markdown/docs/07 extensions/01 Get started/04 Create/02 Output Extensions.md [hunk 1]
```diff
@@ -208,12 +209,12 @@ Now we can use the extension with a test script.
...
 
   ```bash
-  $ ./k6 run test.js -o logger --quiet --no-summary --iterations 2
+  $ ./k6 run test.js --out logger --quiet --no-summary --iterations 2
   ```
 
 <Blockquote mod="note" title="">
 
-The `-o logger` argument tells k6 to use your custom output. The flag
+The `-out logger` argument tells k6 to use your custom output. The flag
```
Comment (javaducky) [inline src/data/markdown/docs/07 extensions/01 Get started/04 Create/02 Output Extensions.md:217]:
```suggestion
The `--out logger` argument tells k6 to use your custom output. The flag
```

**Review by javaducky — CHANGES_REQUESTED** [review]
Just a minor update.

**Comment by javaducky** [comment]
Also, wondering if we shouldn't change the phrasing of the 3rd bullet in https://mdr-ci.staging.k6.io/docs/refs/pull/924/merge/extensions/#extension-use-cases ?

File: src/data/markdown/docs/07 extensions/01 Get started/04 Create/02 Output Extensions.md (hunk 1 above)
Comment (MattDodsonEnglish) [reply src/data/markdown/docs/07 extensions/01 Get started/04 Create/02 Output Extensions.md:217]: :bow:

**Comment by MattDodsonEnglish** [comment]
> Also, wondering if we shouldn't change the phrasing of the 3rd bullet in https://mdr-ci.staging.k6.io/docs/refs/pull/924/merge/extensions/#extension-use-cases ?

Good point, sorry I missed that one. What do you think of the text in https://github.com/grafana/k6-docs/pull/924/commits/42eefe6a8d5a88aeed25131168d44656ee57cf0a?

**Review by javaducky — APPROVED** [review]
LGTM!

**Pull request merged**